README.md

```diff
@@ -5,4 +5,103 @@
 ![Release](https://img.shields.io/github/v/release/instructlab/eval)
 ![License](https://img.shields.io/github/license/instructlab/eval)
 
-Python library for Evaluation
+Python Library for Evaluation
+
+## MT-Bench / MT-Bench-Branch Testing Steps
+
+```shell
+# Optional: Use cloud-instance.sh to launch and setup the instance
+./cloud-instance.sh ec2 launch -t g5.4xlarge
+./cloud-instance.sh ec2 setup-rh-devenv
+./cloud-instance.sh ec2 install-rh-nvidia-drivers
+./cloud-instance.sh ec2 ssh sudo reboot
+./cloud-instance.sh ec2 ssh
+
+
+# Regardless of how you setup your instance
+git clone https://github.com/instructlab/taxonomy.git && pushd taxonomy && git branch rc && popd
+git clone --bare https://github.com/instructlab/eval.git && git clone eval.git/ && cd eval && git remote add syncrepo ../eval.git
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+pip install -e .
+pip install vllm==0.3.3
+python -m vllm.entrypoints.openai.api_server --model instructlab/granite-7b-lab --tensor-parallel-size 1
+```
+
+In another shell window
+
+```shell
+OPENAI_API_KEY="NO_API_KEY" python3 tests/test_gen_answers.py
```

Comment:
@danmcp what do you think about just setting this environment variable in `gen_answers()` and `judge_answers()` in the `MTBenchEvaluator` class? This shouldn't block merging this PR though.